.vscode/settings.json

```diff
@@ -2,5 +2,6 @@
   "jest.pathToJest": "npm run jest --",
   "editor.rulers": [
     80
-  ]
+  ],
+  "javascript.format.enable": false
```

Comment:
Sorry! I added this so the flow validations, don't interfere with the typescript ones. Will remove it.